src/customize-pop/gnome-tweaks-extensions.md

```diff
@@ -0,0 +1,202 @@
+| Scaling Factor | Enable text scaling, which increases font size for HiDPI displays while retaining text sharpness. |
+
+### Keyboard & Mouse
+
+`Keyboard & Mouse` settings extend customization of keyboard and mouse input behavior.
+
+![Keyboard Mouse Tweaks](/images/gnome-tweaks-extensions/keyboard-mouse-tweaks.png)
+
+#### Keyboard
+
+| Option | Function |
+|--------|----------|
+| Show Extended Input Sources | Display additional number of input sources found in the Settings application. |
+| Emacs Input | Apply Emacs keybindings throughout the operating system. |
+| Overview Shortcut | Choose whether left or right `SUPER` keys start the Launcher. |
+
+#### Mouse
+
+| Option | Function |
+|--------|----------|
+| Acceleration Profile | Choose how mouse movement speed adapts to user gestures. |
+| Pointer Location | Pressing the `Ctrl` key creates a visual notification at the mouse pointer location. |
+| Middle Click Paste | Clicking the middle mouse button pastes content from the clipboard. |
+| (Touchpad) Disable While Typing | The touchpad is disabled when using the keyboard. |
+| (Touchpad) Mouse Click Emulation | Select touchpad input that will be interpreted as input from a standard mouse: <ul><li>Fingers: Click with two fingers on the touch pad to input right-click, and three fingers input a middle-click.</li><li>Area: Click the bottom right of the touch pad to input right-click, and the bottom middle for a middle-click.</li><li>Disabled: The touch pad input will not be interpreted as right, middle, or left-click actions. |
+
+### Startup Applications
+
+Choose applications that will automatically start after you log in.
+
+![Startup App Tweaks](/images/gnome-tweaks-extensions/startup-app-tweaks.png)
+
+Add an application by clicking the `+`, selecting an application from the list, and then clicking `Add`.
+
+![Select Startup App](/images/gnome-tweaks-extensions/select-startup-app.png)
+
+### Top Bar
+
+`Top Bar` settings allow you to configure the clock to display additional information.
+
+![Top Bar Tweaks](/images/gnome-tweaks-extensions/top-bar-tweaks.png)
+
+You can also choose to display week numbers in the system calendar application.
+
+![Calendar Tweaks](/images/gnome-tweaks-extensions/calendar-tweaks.png)
+
+### Window Titlebars
+
+These settings allow you to interact with windows using mouse clicks. You can also choose available titlebar buttons and their placement. Right, middle, and left-click buttons can each be configured to perform one of the following actions when clicking a window's titlebar:
+
+![Titlebar Tweaks](/images/gnome-tweaks-extensions/titlebar-tweaks.png)
+
+| Option | Function |
+|--------|----------|
+| Lower | A "send to back" action occurs. |
+| Menu | Display a window manipulation menu. |
+| Minimize | Minimize a window.|
+| Toggle Maximize | Maximize a window, the window returns to its previous size on second click. |
```

Comment:
I would replace all of the `a window` occurrences with `the window` in this entire table. The user will know which window is being operated on.

In this sentence, I would change the comma to a semicolon or a period, or reword further to say "or" between the different possible outcomes instead of referring to a second click specifically (since there could be a third click, fourth click, etc.)